TerrariaServerAPI/TerrariaApi.Server/EventArgs/GrassSpreadEventArgs.cs

```diff
@@ -9,6 +10,6 @@ public class GrassSpreadEventArgs : HandledEventArgs
 		public int Dirt { get; set; }
 		public int Grass { get; set; }
 		public bool Repeat { get; set; }
-		public byte Color { get; set; }
+		public TileColorCache Color { get; set; }
```

Comment:
Can you document this in the tshock changelog in the associated PR since it changes behavior?

Reply:
Good pickup, should be in: https://github.com/Pryaxis/TShock/pull/2670/commits/24c0f106b85ae5e8f039e82047975d3f8c8ad3f1